Counter.py

```diff
@@ -26,14 +26,14 @@ def __init__(self):
 
     def increment(self):
         x = int(self.lbl.text())
-        if x == 33 and self.inc_button.text() == "سبحان الله":
+        if x == 33:
             x = -1
-            self.inc_button.setText("الحمد لله")
-        elif x == 33 and self.inc_button.text() == "الحمد لله":
-            x = -1
-            self.inc_button.setText("الله اكبر")
-        elif x == 33 and self.inc_button.text() == "الله اكبر":
-            x = -1
-            self.inc_button.setText("سبحان الله")
-            self.inc_button.setEnabled(False)
+            match self.inc_button.text():
+                case "سبحان الله":
```

Comment:
What about if we want to have 10+ sentences
It will require increasing the number of cases with repeated code logic, so I suggest putting all Zekr sentences in an array and making the logic code not repeated